lib/page/questions/widget/question_page_widget.dart

```diff
@@ -62,10 +62,17 @@ class QuestionsItemWidget extends StatelessWidget {
                       ? RoundedButtonWidget(
                           buttonText:
                               AppLocalizations.of(context)!.questionsSubmit,
-                          onPressed: () => onButtonPressed.call(),
+                          onPressed: () {
+                            // TODO: Submit answers
+                          },
                         )
                       : NextButtonWidget(
-                          onPressed: () => onButtonPressed.call(),
+                          onPressed: () =>
+                              questionsPageController.animateToPage(
+                            questionNumber,
+                            duration: const Duration(milliseconds: 400),
+                            curve: Curves.easeInOut,
+                          ),
```

Comment:
Nope, we should not pass `PageController` into a child page widget. We should expose 2 callbacks up to `QuestionsPageViewWidget` instead. The previous code we have 1 `onButtonPressed`, having 2 callbacks is good enough.

Reply:
@chornerman 1 issue left 🧑‍🍳